frontend/catalyst/ag_primitives.py

```diff
@@ -26,6 +27,9 @@
 # as well as various utility objects.
 import pennylane as qml
 import tensorflow.python.autograph.impl.api as tf_autograph_api
+from jax.numpy import logical_and as jax_logical_and
+from jax.numpy import logical_not as jax_logical_not
+from jax.numpy import logical_or as jax_logical_or
```

Comment:
The idea is to bind functions early to detect the numpy api change which I think is both safer and easier to read in code (the one-import-per-line style is due to a autoformatter). Why do you think the module-dot-name style is better?

Reply:
Just because that's how we access all functions from the jax numpy api, even across the different PennyLane-ecosystem packages. I'd rather not have another convention that starts with `jax_*`.

Reply:
I would prefer to keep `from module import function` style to bind names early. The `jax_`prefixes are optional, will remove them. Btw I wrote all this in a single line originally, it must be `isort` who made them one import per line for us.

Reply:
> The idea is to bind functions early to detect the numpy api change which I think is both safer and easier to read

I don't see how this detects changes in the api early, could you elaborate?

Reply:
It's extremely unlikely these functions are going to be removed, they are part of the official numpy api.

If we did this for all external functions there would be a lot more import statements across the board, I think the value add here is really low (or even negative). It's also customary across all our code base to use numpy functions as `jnp.*` (and it's variations, `np.` and `pnp.`).

Reply:
Implemented. Although I think that this practice weakens the code-base in general.